<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>D A S H B O A R D</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>JMJ College Library</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="2">
         <widget class="QPushButton" name="deleteRecords">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:#1a69fc;
color:#fff;
border-radius:6px;
}
QPushButton:hover{
background:rgb(0, 255, 0);
color:#000;
}</string>
          </property>
          <property name="text">
           <string>Delete Records</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="borrowNew">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:#1a69fc;
color:#fff;
border-radius:6px;
}
QPushButton:hover{
background:rgb(0, 255, 0);
color:#000;
}</string>
          </property>
          <property name="text">
           <string>Issue a Book</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="ReturnBook">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:#1a69fc;
color:#fff;
border-radius:6px;
}
QPushButton:hover{
background:rgb(0, 255, 0);
color:#000;
}</string>
          </property>
          <property name="text">
           <string>Return Book</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="DeepSearch">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:#1a69fc;
color:#fff;
border-radius:6px;
}
QPushButton:hover{
background:rgb(0, 255, 0);
color:#000;
}</string>
          </property>
          <property name="text">
           <string>Deep Search</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="history">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:#1a69fc;
color:#fff;
border-radius:6px;
}
QPushButton:hover{
background:rgb(0, 255, 0);
color:#000;
}</string>
          </property>
          <property name="text">
           <string>Borrow History</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="studentRegistry">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:#1a69fc;
color:#fff;
border-radius:6px;
}
QPushButton:hover{
background:rgb(0, 255, 0);
color:#000;
}</string>
          </property>
          <property name="text">
           <string>Student Registry</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="bookRegistry">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:#1a69fc;
color:#fff;
border-radius:6px;
}
QPushButton:hover{
background:rgb(0, 255, 0);
color:#000;
}</string>
          </property>
          <property name="text">
           <string>Book Registry</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="backup_Btn">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:#1a69fc;
color:#fff;
border-radius:6px;
}
QPushButton:hover{
background:rgb(0, 255, 0);
color:#000;
}</string>
          </property>
          <property name="text">
           <string>Backup</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="restore_Btn">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:#1a69fc;
color:#fff;
border-radius:6px;
}
QPushButton:hover{
background:rgb(0, 255, 0);
color:#000;
}</string>
          </property>
          <property name="text">
           <string>Restore</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="refresh_Btn">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:#1a69fc;
color:#fff;
border-radius:6px;
}
QPushButton:hover{
background:rgb(0, 255, 0);
color:#000;
}</string>
          </property>
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:#0434EC;
border-radius:6px</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>150</width>
             <height>75</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#fff;</string>
           </property>
           <property name="text">
            <string>Total Books</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="book_count">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>75</y>
             <width>150</width>
             <height>75</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#fff;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:#0434EC;
border-radius:6px</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QLabel" name="label_27">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>150</width>
             <height>75</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#fff;</string>
           </property>
           <property name="text">
            <string>Total Issues</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="issue_count">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>75</y>
             <width>150</width>
             <height>75</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#fff;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:#0434EC;
border-radius:6px</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QLabel" name="label_25">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>150</width>
             <height>75</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#fff;</string>
           </property>
           <property name="text">
            <string>Total Returned</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="return_count">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>75</y>
             <width>150</width>
             <height>75</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#fff;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:#0434EC;
border-radius:6px</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QLabel" name="label_29">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>150</width>
             <height>75</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#fff;</string>
           </property>
           <property name="text">
            <string>Not Returned</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="not_return_count">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>75</y>
             <width>150</width>
             <height>75</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#fff;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
